firmware/Components.cpp

```diff
@@ -543,8 +460,8 @@ void AtlasPH::onSerial(){
   }
 }
 
-AtlasDO::AtlasDO(int channel) 
-  : Sensor(channel), SerialSensor(channel, 9600), measurementInterval(3000)
+AtlasDO::AtlasDO(int id, int port) 
+  : ExternalSensor(id, port), SerialSensor(id, port, 9600), measurementInterval(3000)
```

Comment:
Yea makes sense. I would do a centralized measurementInterval but each sensor should have different default values. Also a different range of acceptable values.